pax/plugins/io/WaveformSimulator.py

```diff
@@ -118,9 +118,25 @@ def s2(self, electrons, g4_id=-1, t=0., x=0., y=0., z=0.):
             x = r * np.cos(phi)
             y = r * np.sin(phi)
 
-        return self.simulator.queue_signal(photon_times, x, y,
+        arriving_photon_times_top,  arriving_photon_times_bottom = self.simulator.queue_signal(photon_times, x, y,
                                            # Generate S2 hitpattern "at the anode": cue for simulator to use S2 LCE map
                                            z=-self.config['gate_to_anode_distance'])
+        peak_top_fraction = float(len(arriving_photon_times_top)) / float(
+            len(arriving_photon_times_bottom) + len(arriving_photon_times_top) )
+        self.store_true_peak('s2',
+                                          g4_id,
+                                          t,
+                                          x,
+                                          y,
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->